src/extension.ts

```diff
@@ -85,6 +86,20 @@ export async function activate(context: vscode.ExtensionContext) {
 					}
 					await TerminalCommandExecuter.runCommand(nodeVersionCommand, terminalTitle, terminalIcon);
 				}
+
+				//Look for the custom steps string in the project file and run the command if found
+				if (fileContents.indexOf(ProjectFileContent.installCustomSteps) > -1) {
+					const value = getExtensionSettings<string>('projectCustomSteps', '');
+					if (value) {
+						const jsonArray = JSON.parse(value);
+						// eslint-disable-next-line no-console
+						console.log(jsonArray.length);
+						for (let i = 0; i < jsonArray.length; i++) {
+							const item = jsonArray[i];
+							await TerminalCommandExecuter.runCommand(item.command, terminalTitle, terminalIcon);
```

Comment:
let's consolidate this to a single line. If `item` is used in a single place the there is little benefit to extract it to a separate variable

```suggestion
							await TerminalCommandExecuter.runCommand(jsonArray[i].command, terminalTitle, terminalIcon);
```